### ISSUE TITLE
Some questions about session recordings

### ISSUE THREAD

**Issue by geanferrani123** [Ylianst/MeshCentral#956]
Guys, good morning!

I'm not able to find the player.htm inside the meshcentral, I need to see a recording, where is it now? I noticed that this happens from version 0.4.8-u.

Another topic, can I convert the mcrec file to other extensions? Like for example, mp4, mpeg and etc?

**Comment by Ylianst** [comment]
You should be able to access it by adding /player.htm to the MeshCentral URL, but only once you have logged into the server. For example, once logged into [MeshCentral.com](https://www.meshcentral.com), you can access it here: [https://meshcentral.com/player.htm](https://meshcentral.com/player.htm).

As for converting the file, there is no way to do that right now.I guess you could play the file and use a screen recorder.

**Comment by geanferrani123** [comment]
> You should be able to access it by adding /player.htm to the MeshCentral URL, but only once you have logged into the server. For example, once logged into [MeshCentral.com](https://www.meshcentral.com), you can access it here: https://meshcentral.com/player.htm.
> 
> As for converting the file, there is no way to do that right now.I guess you could play the file and use a screen recorder.

Hi Ylianst,

Yeah, I add /player.htm and don't access it.

Going to / etc / meshcentral / node_modules / meshcentral / public note that the file is not there, update another server that is running meshcentral for testing and update not sent as well.

Can you help me?

![image](https://user-images.githubusercontent.com/55212206/74852041-d4a41680-531a-11ea-85a2-5f2d6e5ddd6b.png)

**Comment by geanferrani123** [comment]
When I send the file manually, the following error occurs:

![image](https://user-images.githubusercontent.com/55212206/74852514-70ce1d80-531b-11ea-95fc-e78bafa6b6d3.png)

![image](https://user-images.githubusercontent.com/55212206/74852535-79265880-531b-11ea-9d08-367fff4709e6.png)

**Comment by MailYouLater** [comment]
I said this last time someone had this problem, but it looks like it still didn't get fixed.  Your not getting redirected from http to https.  See the difference between http://meshcentral.com/player.htm and https://meshcentral.com/player.htm

**Comment by Ylianst** [comment]
Going to close this as the new player is not a .handlebar file and handled differently. Please open a new issue if this is a problem again.
